controllers/discovery/initDiscoveryDomainData.js

```diff
@@ -14,6 +14,12 @@ const InitDiscoveryDomainData = async (req, res) => {
   const userId = req.userId;
 
   try {
+    let totalDataQuery = `SELECT 
+                              count(A.domain_page_id) as total_data
+                          FROM domain_page A 
+                          WHERE A.credder_score >= ${CREDDER_MIN_SCORE} AND A.status = true
+                          `;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The SQL query is directly using the `CREDDER_MIN_SCORE` variable in the string, which could potentially lead to SQL injection if the value of `CREDDER_MIN_SCORE` is not properly sanitized or controlled. It's better to use parameterized queries or prepared statements to prevent this security issue.
```diff
- WHERE A.credder_score >= ${CREDDER_MIN_SCORE} AND A.status = true
+ WHERE A.credder_score >= :minScore AND A.status = true
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->